.github/workflows/windows-qe-tpl.yml

```diff
@@ -104,6 +106,9 @@ jobs:
             --project-name 'windows-desktop-${{ matrix.windows-version }}-${{ matrix.windows-featurepack }}-${{inputs.qe-type}}-${{inputs.preset}}' \
             --backed-url azblob://crc-qenvs-state/${{ github.repository }}-${{ github.run_id }} \
             --conn-details-output '/workspace' \
+            --nested-virt \
+            --memory ${{matrix.memory }} \
```

Comment:
You can put values directly here...matrix is fine but it is more intended for multiple values for a field...when you have just one..you can set it directly on the code (or variable, but not matrix)

Reply:
as those are the default values, I'm unsure what's the difference setting it explicitely. Not setting them those are the Implicit values

Reply:
actually you would not need them...as you said those are already default...the only one which you need and it is not default is `nested-virt` so it is really up to you even just remove them and leave only `nested-virt`